hack/kwok/stages/pod-unhealthy.yaml

```diff
@@ -0,0 +1,68 @@
+apiVersion: kwok.x-k8s.io/v1alpha1
```

Comment:
Why put this in the hack and not make this part of the Kwok package? e.g. `kwok/stages` or even make it part of the chart directly? I don't really see this as a hack? Also, is this something that we built or something that's available upstream from kwok?

Reply:
This is not a hack, and the way that KWOK recommends to mock pod and node health: https://kwok.sigs.k8s.io/docs/user/stages-configuration/. This seemed reasonable to me at the time, but I agree the chart is a better place to add this CR. This was inspired based from files available from upstream KWOK. However these is a CR we needed to build